src/applications/healthcare/questionnaire/containers/QuestionnaireWrapper.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import {
+  selectShowQuestionnaire,
+  selectLoadingFeatureFlags,
+} from '../selectors';
+
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import App from './App';
+
+const QuestionnaireWrapper = ({
+  location,
+  children,
+  isQuestionnaireEnabled,
+  isLoadingFeatureFlags,
+}) => {
+  if (isLoadingFeatureFlags) {
+    return (
+      <>
+        <LoadingIndicator />
+      </>
+    );
+  } else if (!isQuestionnaireEnabled) {
+    window.location.replace('/');
+    return <></>;
```

Comment:
Just out of curiosity why do we want to return a empty fragment rather than just `null`?

Reply:
cypress needed to return something not falsy for the test